README.md

```diff
@@ -621,13 +621,38 @@ npm install prosemirror-utils
    ```
 
 
- * **`cloneRowAt`**`(cloneRowIndex: number) → fn(tr: Transaction) → Transaction`\
-   Returns a new transaction that adds a new row after `cloneRowIndex`, cloning the row attributes at `cloneRowIndex`.
+ * **`copyRow`**`(insertNewRowIndex: number, rowToBeClonedIndex: number, options: ?CopyRowOptions) → fn(tr: Transaction) → Transaction`\
```

Comment:
can we just add an optional parameter to `cloneRowAt` that'd control the behaviour of copying attributes?

Reply:
We can do it, but I don't think that option is the best one for it, because the cloneRowAt and copyRow are totally different.

Reply:
`cloneRowAt` always copy de previous row `copyRow` is flexible and you should explict says which row you are copying and it place you are going to insert it

Reply:
does it make sense to copy anything other than the previous row? Can you provide a use case for that?

Reply:
if you need to copy programmatically a specific row to add on the line above, not bellow.If we keep the only-the-previous we will be limiting this function.

Reply:
I'd avoid doing it as much as possible, here's why:
* the cost of supporting an API that doesn't serve a specific purpose is very high. We shouldn't expose some functionality as public API just because we can, it needs to solve a problem. We don't have that problem and I didn't see anyone requesting it as a feature.
* no RFCs for deprecating existing APIs and no further no breaking changes required
* a simple optional parameter would do everything we need. If we want to deprecate anything in the future, we could do that separately.
* fewer tests to support